File: 2st/Yeonwoo/세로 읽기.py
def solution(my_string, m, c):
    answer = ''
    temp = []
    for i in range(len(my_string)//m):
        temp.append(my_string[m*i:m*(i+1)])
    for i in range(len(temp)):
        answer+= temp[i][c-1]
    return answer

File: 3st/Juhee/세로 읽기.java
class Solution {
    public String solution(String my_string, int m, int c) {
        String answer = "";
        for(int i = 0 ; i < my_string.length() ; i++){
            if( (i+1) % m == c || m == 1 || ((i+1) % m == 0 && c == m) ){
                answer += my_string.charAt(i);
            }
        }
        return answer;
    }